js/source/esri-vector-tile-source.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var util = require('mapbox-gl/js/util/util');
```

Comment:
Ahh yes, the way the code was written and built was done outside of the actual code base, as a sort of wrapper. This file is now removed.